vendor/assets/javascripts/jquery.autoSuggest.custom.js

```diff
@@ -46,7 +46,7 @@
 		  	selectionRemoved: function(elem){ elem.remove(); },
 		  	formatList: false, //callback function
 		  	beforeRetrieve: function(string){ return string; },
-		  	retrieveComplete: function(data){ return data; },
+		  	retrieveComplete: function(data){ return sortArrayOfObj(data, "name"); },
```

Comment:
This seems like a unnecessary modification, please overwrite it on the library calls instead.